ee/api/session_summaries.py

```diff
@@ -93,8 +93,18 @@ def _validate_input(self, request: Request) -> tuple[list[str], datetime, dateti
             extra_summary_context = ExtraSummaryContext(focus_area=focus_area)
         return session_ids, min_timestamp, max_timestamp, extra_summary_context
 
-    def _if_video_validation_enabled(self, user: User) -> bool | None:
-        # Check if the summaries should be validated with videos
+    def _determine_video_validation_enabled(self, user: User) -> bool | Literal["full"] | None:
```

Comment:
Don't like 3 return types even one bit, but understand the reasoning, ok to keep it.